code-examples/php/usage-examples/bulkWrite.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require 'vendor/autoload.php'; 
+
+// Replace the uri string with your MongoDB deployment's connection string.
+$uri = "mongodb://localhost:27017";
+
+$collection = (new MongoDB\Client($uri))->sample_mflix->movies;
+
+try {
```

Comment:
I'm not sure the `try/catch` is required at all. We don't even bother with this in our own examples (e.g. [bulk.php](https://github.com/mongodb/mongo-php-library/blob/master/examples/bulk.php)).